boards/arm/nucleo_l011k4/arduino_r3_connector.dtsi

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2020 Steven Daglish <s.c.daglish@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	arduino_header: connector {
+		compatible = "arduino-header-r3";
+		#gpio-cells = <2>;
+		gpio-map-mask = <0xffffffff 0xffffffc0>;
+		gpio-map-pass-thru = <0 0x3f>;
+		gpio-map = <0 0 &gpioa 0 0>,	/* A0 */
+			   <1 0 &gpioa 1 0>,	/* A1 */
+			   <2 0 &gpioa 3 0>,	/* A2 */
+			   <3 0 &gpioa 4 0>,	/* A3 */
+			   <4 0 &gpioa 5 0>,	/* A4 */
+			   <5 0 &gpioa 6 0>,	/* A5 */
+			   <5 0 &gpioa 7 0>,	/* A6 */
+			   <5 0 &gpioa 2 0>,	/* A7 */
+			   <6 0 &gpioa 10 0>,	/* D0 */
+			   <7 0 &gpioa 9 0>,	/* D1 */
+			   <8 0 &gpioa 12 0>,	/* D2 */
+			   <9 0 &gpiob 0 0>,	/* D3 */
+			   <10 0 &gpiob 7 0>,	/* D4 */
+			   <11 0 &gpiob 6 0>,	/* D5 */
+			   <12 0 &gpiob 1 0>,	/* D6 */
+			   <13 0 &gpioc 14 0>,	/* D7 */
+			   <14 0 &gpioc 15 0>,	/* D8 */
+			   <15 0 &gpioa 8 0>,	/* D9 */
+			   <16 0 &gpioa 11 0>,	/* D10 */
+			   <17 0 &gpiob 5 0>,	/* D11 */
+			   <18 0 &gpiob 4 0>,	/* D12 */
+			   <19 0 &gpiob 3 0>;	/* D13 */
+	};
```

Comment:
You're describing arduino V3 connector, but board provides arduino Nano.
There might be some compatibility but aim of "connectors" is to provide electronic and physical compatibility, which is not the case here.

Reply:
I need to look into this further. The pins above are for the Arduino Nano compatibility as written on the Nucleo board case.

Reply:
I have removed all reference to Arduino pins.